storage/doc.go

```diff
@@ -349,11 +347,12 @@ To create a client which will use gRPC, use the alternate constructor:
 	}
 	// Use client as usual.
 
-Using the gRPC API inside GCP with a bucket in the same region can allow for
-Direct Google Access (enabling requests to skip some proxy steps and reducing
-response latency). A warning is emmitted if gRPC is not used within GCP to
-warn that Direct Google Access could not be initialized. Direct Google Access
-is not required to access the gRPC API.
+Using the gRPC API inside GCP with a bucket in the same region can allow direct
+connectivity (enabling requests to skip Google Front Ends (GFEs) and reducing
```

Comment:
Google Front Ends is another term without public docs, I would revert to the original wording here and just add the link for the (new?) DP docs. No need to duplicate more information here.